secretmanager/snippets/regional_samples/enable_regional_secret_version.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+command line application and sample code for enabling a regional
+secret version.
+"""
+
+import argparse
+
+from google.cloud import secretmanager_v1
+
+
+# [START secretmanager_enable_regional_secret_version]
+def enable_regional_secret_version(
+    project_id: str, location_id: str, secret_id: str, version_id: str
+) -> secretmanager_v1.EnableSecretVersionRequest:
+    """
+    Enable the given secret version, enabling it to be accessed after
+    previously being disabled. Other secrets versions are unaffected.
+    """
```

Comment:
suggestion: can you provide a docstring explaining the arguments of this function?